### PR TITLE
Applies table exact widths in expanded view according to design

### PR DESCRIPTION
**Pull request by yuvallevanoni**
- Adds paddings + widths + maxWidths
- Fixes sidebar width

### CODE REVIEW

File: src/components/common/Logo/style.module.scss [hunk 1]
```diff
@@ -1,5 +1,5 @@
 .logo {
-  padding: 18px 0 0 21px;
+  padding: 18px 0 0 7px;
```
Comment (yuvallevanoni) [inline src/components/common/Logo/style.module.scss:2]: This change is so the overall width of the table will be 1760px according to design (and entire screen is 1920px).
The sidebar itself should change in a different PR, so this is only a minor adaptation

File: src/styles/variables.scss [hunk 2]
```diff
@@ -1,6 +1,6 @@
 :root {
   --desktop-min-width: 1280px;
-  --sidebar-width: 110px;
```
Comment (yuvallevanoni) [inline src/styles/variables.scss:3]: This change is so the overall width of the table will be 1760px according to design (and entire screen is 1920px).
The sidebar itself should change in a different PR, so this is only a minor adaptation

File: src/components/common/Logo/style.module.scss (hunk 1 above)
Comment (dr15) [reply src/components/common/Logo/style.module.scss:2]: This will change the logo for the old design too. Is that what you were going for?

**Review by dr15 — CHANGES_REQUESTED** [review]
I checked it on my screen which is 1920px and there is a horizontal scrollbar that I think should not be there?

File: src/components/Layout/Sidebar-v2/Navigation/Navigation.test.tsx [hunk 3]
```diff
@@ -0,0 +1,44 @@
+import React from 'react';
```
Comment (yuvallevanoni) [inline src/components/Layout/Sidebar-v2/Navigation/Navigation.test.tsx:1]: Exact copy from the Sidebar.
Only Sidebar-v2/style.module.scss has changed

File: src/components/Layout/Sidebar-v2/style.module.scss [hunk 4]
```diff
@@ -0,0 +1,46 @@
+.sidebar {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  width: var(--sidebar-width-v2);
```
Comment (yuvallevanoni) [inline src/components/Layout/Sidebar-v2/style.module.scss:5]: First change

File: src/components/Layout/Sidebar-v2/style.module.scss [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+
+  .sidebar-separator:global(.ant-divider) {
+    margin: 5px 19px 15px 19px;
+    width: calc(100% - 38px);
+    min-width: calc(100% - 38px);
+  }
+}
+
+.sidebar-skeleton {
+  width: var(--sidebar-width-v2);
```
Comment (yuvallevanoni) [inline src/components/Layout/Sidebar-v2/style.module.scss:23]: second change

File: src/components/common-v2/Logo/style.module.scss [hunk 6]
```diff
@@ -0,0 +1,18 @@
+.logo {
+  padding: 18px 0 0 7px;
```
Comment (yuvallevanoni) [inline src/components/common-v2/Logo/style.module.scss:2]: Changed from 21px to 7px for padding-left

**Review by yuvallevanoni — COMMENTED** [review]
I created new v2 folders for:
- layout -> sidebar
- layout -> headerToolBar

This can be used as the root for later tickets regarding the new design for these layout components

**Pull request closed without merging**